scan/spec/detect_values_spec.rb

```diff
@@ -3,13 +3,15 @@
     describe 'Xcode config handling' do
       before do
         options = { project: "./scan/examples/standard/app.xcodeproj" }
+        FileUtils.mkdir_p("./scan/examples/standard/app.xcodeproj/project.xcworkspace/xcuserdata/#{ENV['USER']}.xcuserdatad/")
+        FileUtils.copy("./scan/examples/standard/WorkspaceSettings.xcsettings", "./scan/examples/standard/app.xcodeproj/project.xcworkspace/xcuserdata/#{ENV['USER']}.xcuserdatad/WorkspaceSettings.xcsettings")
```

Comment:
Is this `ENV['USER`]` needed in these tests? Is it possible for us to put a static path there instead?

Reply:
These settings can't be in the shared xcsettings, so we need to use the current user.

Reply:
👌